features/filesystem/bd/BufferedBlockDevice.cpp

```diff
@@ -36,6 +36,10 @@ BufferedBlockDevice::~BufferedBlockDevice()
 
 int BufferedBlockDevice::init()
 {
+    if (_init_ref_count++) {
```

Comment:
ref_count operation do not need to be atomic?

Reply:
Absolutely right. Fixed now.